_entries/2016-05-04-hyperlinks.md

```diff
@@ -35,16 +35,20 @@ Not this
 
 Don’t use meaningless terms such as ‘click here’, ‘read more’ or ‘useful links’.
 
-### Drop the http://
+### Drop the https://
 
-When referencing a website, drop the `http://` prefix.
+You don't need to write `http://` or `https://` when referencing a website.
+
+If the site has a URL route without `www` you don't need to write it when you reference it. But keep the `www` if there is only a redirect in place.
+
+You still need to use the full URL when you create a hyperlink.
```

Comment:
I wonder if we could we be less prescriptive about the technical detail and simply say "Regardless of the text used when linking, you should test that the link works when accessed."? i.e. it's a bit absurd to link to the incorrect destination because the style guide told you to :) Also conscious that some users will be working in content management systems that might abstract away the actual href in the editing interface, which makes validation per 'full URL' difficult.

There is maybe some more guidance needed here around how to check a link is working! For example, content should never link to an environment that's not available to the intended audience – it’s easy as an author to accidentally paste an internal CMS link into an external facing product (I've seen this often in large media organisations; and anecdotally observed on government websites). At a minimum this may mean authors validate content that is sent live is working as intended, but could also include logging out of a CMS in pre-production, if possible, to check links (and images/all linked media really!) are reachable. The latter might be a little too specific for the guide though…